tensorflow/workspace.bzl

```diff
@@ -52,6 +52,30 @@ temp_workaround_http_archive = repository_rule(
       "strip_prefix": attr.string(default = ""),
    })
 
+def _repos_are_siblings():
+  return Label("@foo//bar").workspace_root.startswith("../")
```

Comment:
FYI: This function might need to be updated too: https://github.com/bazelbuild/rules_closure/blob/master/closure/private/defs.bzl#L135